src/views/Settings.vue

```diff
@@ -68,12 +68,10 @@ export default defineComponent({
   },
   methods: {
     setFacility (facility: any) {
-      this.userProfile.facilities.map((fac: any) => {
-        if (fac.facilityId == facility['detail'].value) {
-          this.store.dispatch('user/setFacility', {'facility': fac});
-          console.log(fac);
-        }
-      })
+      if (this.userProfile)
```

Comment:
I think we should not skip `{}` for better readability

Reply:
Improved code readability.
Thank you sir for reviewing my code.